### DISCUSSION TITLE
[v3 beta] Delete unused panel provider, and getting errors file not found

### DISCUSSION THREAD

**Discussion by akunbeben** [filamentphp/filament#7079] [Help] [answered]
![image](https://github.com/filamentphp/filament/assets/46495960/e03bca88-45c0-47b5-98be-137ff38b1323)

I've playing around with v3. I just created 2 panels, Admin & App panel. 

It is working properly, then I delete the AppPanelProvider to remove the panel, and that error appears and I thought it was discovering panels automatically. How can I discover/register the panels manually?

**Comment by mstfkhazaal** [comment]
try composer dump-autoload

**Comment by akunbeben** [comment, marked as answer]
Just realized that `make:filament-panel` is literally registering the service provider to the `config/app.php` file. So, I just need to remove it from the config file.

```php
'providers' => ServiceProvider::defaultProviders()->merge([
    ...
    App\Providers\Filament\AppPanelProvider::class,
    App\Providers\Filament\AdminPanelProvider::class,
    App\Providers\Filament\BackofficePanelProvider::class,
    ...
])->toArray(),
```

I think this should be documented. But, I'm not sure the right place to put this on the documentation.

**Reply by akunbeben** [reply]
https://beta.filamentphp.com/docs/3.x/panels/configuration/#creating-a-new-panel

Oh, already on the documentation, my bad!
